artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/Tomcat_9-Configure-w1/ArtifactTemplate.tosca

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions targetNamespace="http://opentosca.org/artifacttemplates" id="winery-defs-for_artifacttemplates-Tomcat_9-Configure-w1" xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" >
```

Comment:
Replace "winery-defs-for_" --> "" for all Ids to confirm to the new Id convention?